airgun/views/cloud_inventory.py

```diff
@@ -59,15 +60,15 @@ class uploading(InventoryTab):
     @property
     def is_generating(self):
         try:
-            self.parent_browser.selenium.find_element_by_xpath(f'{self.STATUS_ELEMENTS}[1]/span')
```

Comment:
I think you should use `self.parent_browser.element()` here, instead of dropping into selenium directly. 

The function is defined in the wt.core browser class and will accept a locator string.

Reply:
ok will check that!